CREDITS-LEVELS

```diff
@@ -0,0 +1,126 @@
+--Phase 1--
+
+Episode 1 - Outpost Outbreak
+    E1M1: Outer Prison by Gabriel "YukiHerz" Antonio
+    E1M2: Communications Center by acc
+    E1M3: Waste Disposal by Josef "Paar" Šustek
+        E1M9: Nutrient Recycling by Mortrixs19
+    E1M4: Supply Depot by GeekMarine
+    E1M5: Armory by Oleg "Wereknight" Vovk
+    E1M6: Training Facility by Luiz Henrique "Xindage" Gasparin Jeronimo
+    E1M7: Xenobiotic Materials Lab by Will "Kid Airbag" Hackney
+    E1M8: Outpost Quarry by Luiz Henrique "Xindage" Gasparin Jeronimo
+
+Episode 2 - Military Labs
+    E2M1: Elemental Gate by Luiz Henrique "Xindage" Gasparin Jeronimo and m
+    E2M2: Shifter by Hayden49
+    E2M3: Reclaimed Facilities by SUNPYG Senpai
+    E2M4: Flooded Installation by Luiz Henrique "Xindage" Gasparin Jeronimo
+    E2M5: Underground Hub by Luiz Henrique "Xindage" Gasparin Jeronimo
+        E2M9: Fortress 31 by Miguel "Z0k" Suarez Gomez
+    E2M6: Hidden Sector  by Andrew "Catoptromancy" Wood
+    E2M7: Control Complex by Luiz Henrique "Xindage" Gasparin Jeronimo
+    E2M8: Containment Cell by Luiz Henrique "Xindage" Gasparin Jeronimo
+
+Episode 3 - Event Horizon
+    E3M1: Land of the Lost by Jay "Jayextee" Trent
+    E3M2: Geothermal Tunnels by pan-te
+    E3M3: Sacrificial Bastion by Jeremy "SgtCrispy" Elder
+    E3M4: Oblation Temple by Mortrixs19 and Erick "GuyNamedErick" Tenorio
+    E3M5: Infernal Hallows by SUNPYG Senpai
+    E3M6: Igneous Intrusion by Archfile
+        E3M9: Acquainted With Grief by Angry Saint
+    E3M7: No Regrets by Robin "Hawkwind" Palmer
+    E3M8: Ancient Lair by Kim "Torn" Bach
+
+Episode 4 - Double Impact
+    E4M1: Maintenance Area by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+    E4M2: Research Complex by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+        E4M9: Operations by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+    E4M3: Central Computing by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+    E4M4: Hydroponic Facility by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+    E4M5: Engineering Station by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+    E4M6: Command Center by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+    E4M7: Waste Treatment by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+    E4M8: Launch Bay by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
+
+
+
```

Comment:
You have three blank lines here, but only two for the start of FreeDM.

Reply:
Fixed.